CRUD/models.py

```diff
@@ -31,27 +34,33 @@ class AdjunctFacultyMember(models.Model):
         ('3', 'step 3'),
         ('F', 'faculty')
     }
+    DOY = ('1980', '1981', '1982', '1983', '1984', '1985', '1986', '1987',
+       '1988', '1989', '1990', '1991', '1992', '1993', '1994', '1995',
+       '1996', '1997', '1998', '1999', '2000', '2001', '2002', '2003',
+       '2004', '2005', '2006', '2007', '2008', '2009', '2010', '2011',
+       '2012', '2013', '2014', '2015')
+
     a_f_eaf_c_crs_list = models.CharField(null=False, blank=False, choices=a_f_eaf_c_crs_choices, max_length=3)
-    semester = models.CharField(max_length=4)
+    semester = models.CharField(max_length=4, help_text="Please enter in format: semester year (ex: FA19)")
     first_name = models.CharField(null=False, blank=False, max_length=30)
     last_name = models.CharField(null=False, blank=False, max_length=30)
-    date_of_birth = models.DateField(null=False, blank=False, default=datetime.now)
-    employeeID = models.IntegerField(null=False, blank=False, unique=True,primary_key=True)
+    date_of_birth = models.DateField(null=False, blank=False)
+    employeeID = models.IntegerField(null=False, blank=False, unique=True, primary_key=True)
     step_rate = models.CharField(null=False, blank=False, choices=sr_choices, max_length=10, default='step 1')
-    I9_completed = models.DateField(null=False, blank=False)
-    I9_greater_than_3_years = models.IntegerField(null=False, blank=False)
+    I9_completed = models.DateField(null=False, blank=False, help_text="Please enter a date.")
+    I9_greater_than_3_years = models.IntegerField(null=False, blank=False, help_text="Please enter a number of years.")
     background_passed = models.CharField(null=False, blank=False, choices=bg_choices, max_length=2)
     cv_resume = models.IntegerField(help_text="please enter a year")
     masters = models.CharField(choices=masters_choices, max_length=3)
-    CTL_notified = models.DateField(null=False, blank=False)
+    CTL_notified = models.DateField(null=False, blank=False, help_text="Please enter a date.")
     address = models.CharField(null=False, blank=False, max_length=50)
     city = models.CharField(null=False, blank=False, max_length=20)
     state = models.CharField(null=False, blank=False, max_length=2)
     zip = models.IntegerField(null=False, blank=False, )
     primary_email = models.EmailField(null=False, blank=False)
     secondary_email = models.EmailField()
-    primary_phone = PhoneField(null=False, blank=False, help_text='Primary phone number')
-    secondary_phone = PhoneField(blank=True, help_text='Secondary phone number')
+    primary_phone = PhoneField(null=False, blank=False)
+    secondary_phone = PhoneField(blank=True, help_text='Optional')
```

Comment:
Rather than having an optional tooltip here all required fields should be marked with a star as standard practice and we should add a required note at the top of the page.

like this: http://i.stack.imgur.com/UtQbD.png 
or this: https://brand-experience.ieee.org/wp-content/uploads/2017/07/required_field.jpg